versions.go

```diff
@@ -0,0 +1,67 @@
+// Copyright 2015 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"strings"
+
+	"github.com/openconfig/goyang/pkg/yang"
+	"github.com/pborman/getopt"
+)
+
+func init() {
+	flags := getopt.New()
+	register(&formatter{
+		name:              "oc-versions",
+		f:                 doOcVersions,
+		includeSubmodules: true,
+		help:              "output files that describe a non-null schema",
+		flags:             flags,
+	})
+}
+
+func doOcVersions(w io.Writer, entries []*yang.Entry) {
+	for _, e := range entries {
+		m, ok := e.Node.(*yang.Module)
+		if !ok {
+			fmt.Fprintf(os.Stderr, "error: cannot convert entry %q to *yang.Module", e.Name)
+			continue
+		}
+
+		fmt.Fprintf(w, "%s.yang:", m.Name)
+
+		for _, e := range m.Extensions {
+			keywordParts := strings.Split(e.Keyword, ":")
+			if len(keywordParts) != 2 {
+				// Unrecognized extension declaration
+				continue
```

Comment:
Like on line 42, could you write a message to `os.Stderr` indicating what format must be used by the user? (help the user to understand their mistake)